config/helm/chart/default/templates/Common/operator/clusterrole-operator.yaml

```diff
@@ -28,6 +28,12 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - pods/log
+    verbs:
+      - get
```

Comment:
since we are only accessing operator components with known names, can this be restricted to specific `resourceNames`?